firefox-ios/Client/TabManagement/Tab+ChangeUserAgent.swift

```diff
@@ -7,11 +7,34 @@ extension Tab {
         // Track these in-memory only
         static var privateModeHostList = Set<String>()
 
+        private static let uaSetFilename = "changed-ua-set-of-hosts.xcarchive"
+
         private static let file: URL = {
-            let root = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0]
-            return root.appendingPathComponent("changed-ua-set-of-hosts.xcarchive")
+            let cache = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask)[0]
+            return cache.appendingPathComponent(uaSetFilename)
         }()
 
+        // Migration function to move the file from documents to cache directory
+        static func migrateFile() {
```

Comment:
Can you add unit test around this?

Reply:
Sure, I'll add unit tests for that. Thanks for the heads up!